manifests/i/InfiniteRed/Reactotron/Beta/3.0.0.1039/InfiniteRed.Reactotron.Beta.locale.en-US.yaml

```
@@ -0,0 +1,33 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: InfiniteRed.Reactotron.Beta
+PackageVersion: 3.0.0.1039
+PackageLocale: en-US
+Publisher: Infinite Red
+PublisherUrl: https://github.com/infinitered
+PublisherSupportUrl: https://github.com/infinitered/reactotron/issues
+#PrivacyUrl: 
+Author: Infinite Red
+PackageName: Reactotron
```

Even tho this is incorrect we need to do this so 2 packages don't have the same PackageName and Publisher
```suggestion
PackageName: Reactotron Beta
```